static/css/section/_mobile.scss

```diff
@@ -1,40 +1,84 @@
-.mobile-features {
-  .row-hero {
-    overflow: hidden;
-
-    @media only screen and (min-width : $breakpoint-medium) {
+.mobile- {
```

Comment:
It's to avoid the repetition. There's no content in the style so the class ".mobile-" never exists in the CSS.

It's a matter of style but I can change it.